starbook-backend/flask_methods.py

```diff
@@ -14,11 +14,15 @@ def __init__(self, api):
 
         @app.route("/")
         def send_index():
-            return send_file('index.html')
+            return render_template('index.html',
+                                   client_id=CLIENT_ID,
+                                   starbook_api=STAR_BOOK_API_ENDPOINT,
+                                   send_cookies=int(SEND_COOKIES),
```

Comment:
why int()?

Reply:
because im doing `Boolean(send_cookies)` on the js side